extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -407,17 +460,20 @@ public static void setNumberGenerator(Random random) {
         numberGenerator = random;
     }
 
-    public static boolean areHardcodedYoutubeMusicKeysValid() throws IOException, ReCaptchaException {
-        final String url = "https://music.youtube.com/youtubei/v1/search?alt=json&key=" + HARDCODED_YOUTUBE_MUSIC_KEYS[0];
+    public static boolean areHardcodedYoutubeMusicKeysValid() throws IOException,
+            ReCaptchaException {
+        final String url =
+                "https://music.youtube.com/youtubei/v1/music/get_search_suggestions?alt=json&key="
+                + HARDCODED_YOUTUBE_MUSIC_KEY[0];
 
         // @formatter:off
         byte[] json = JsonWriter.string()
```

Comment:
This json could also be probably hardcoded in a faster string, but let's keep this to another PR.